.github/workflows/tests.yml

```diff
@@ -76,11 +78,11 @@ jobs:
 
             - name: Check commit message
               env:
-                  BRANCH: ${{ github.event.pull_request.base.ref }}
+                  BRANCH: ${{ github.event.pull_request.base.sha }}
```

Comment:
It might be cleaner to get rid of the env variables. Anyways, I'm approving and giving a chance to Marc to figure this out. Thanks for the help!